tests/drivers/regulator/voltage/dts/bindings/test-regulator-voltage.yaml

```diff
@@ -42,3 +42,17 @@ properties:
     default: 0
     description: |
       Delay between voltage set and ADC read. Defaults to 0 (no delay).
+
+  min-microvolt:
+    type: int
+    default: 0
```

Comment:
volts can take `int32_t` range (includes negatives)

Reply:
Change default to INT32_MIN.